tools/infrasys/src/main.rs

```diff
@@ -16,36 +17,33 @@ use std::collections::HashMap;
 use std::num::NonZeroUsize;
 use std::path::{Path, PathBuf};
 use std::{fs, process};
-use structopt::{clap, StructOpt};
 use tokio::runtime::Runtime;
 use url::Url;
 
 //   =^..^=   =^..^=   =^..^=  SUB-COMMAND STRUCTS  =^..^=   =^..^=   =^..^=
 
-#[derive(Debug, StructOpt)]
-#[structopt(setting = clap::AppSettings::DeriveDisplayOrder)]
+#[derive(Debug, Parser)]
 struct Args {
-    #[structopt(global = true, long, default_value = "INFO")]
+    #[arg(global = true, long, default_value = "INFO")]
     log_level: LevelFilter,
 
-    // Path to Infra.toml  (NOTE: must be specified before subcommand)
-    #[structopt(long, parse(from_os_str))]
+    // Path to Infra.toml (NOTE: must be specified before subcommand)
+    #[arg(long)]
     infra_config_path: PathBuf,
 
-    #[structopt(subcommand)]
-    subcommand: SubCommand,
+    #[command(subcommand)]
+    subcommand: SubCommands,
 }
 
-#[derive(Debug, StructOpt)]
-#[structopt(setting = clap::AppSettings::DeriveDisplayOrder)]
```

Comment:
I had a bit of trouble looking up `DeriveDisplayOrder`.  I'm assuming it doesn't exist now, but are we sure removing it doesn't change behavior?